docs/development.md

```diff
@@ -99,30 +124,100 @@ Regardless of how you choose to run the operator, before doing so ensure the `Up
 $ oc create -f deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml
 ```
 
+MUO by defaults uses in the internal services to contact prometheus and alertmanager. This enables the use of a firewall to prevent egress calls however increases local development complexity slightly. 
+
+There are now three main modes that MUO can be ran in. 
+
+1. Run in a container in cluster. 
+2. Run locally using port-forwards and `/etc/hosts` entries to replicate production environment. 
+3. Run locally using Routes to access services. This is not true production however is the most simple for local development. 
+
+Modes 2 and 3 can be executed via the `Makefile` optionally setting the `$OPERATOR_NAMESPACE` as explored in the next section. 
+
+```
+run                              Wrapper around operator sdk run. Requires OPERATOR_NAMESPACE to be set. See run-standard for defaults.
+run-standard                     Run locally with openshift-managed-upgrade-operator as OPERATOR_NAMESPACE.
+run-routes                       Same as `run`, however will use ROUTE objects to contact prometheus and alertmanager. Use of routes is non-standard but convenient for local development. Requires OPERATOR_NAMESPACE to be set.
+run-standard-routes              Run locally with openshift-managed-upgrade-operator as OPERATOR_NAMESPACE and use of non-standard routes.
+```
+
+
 ### Locally
 
 - Make sure you have the [operator-sdk](#operator-sdk) `$PATH`.
 
-- If you are not using an account that has `cluster-admin` privileges, you will need to [elevate permissions](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/manage-privileges.md) to possess them.
+- You will need to be logged in with an account that meets the [RBAC requirements](https://github.com/openshift/managed-upgrade-operator/blob/master/deploy/cluster_role.yaml) for the MUO service account.
 
+<<<<<<< Updated upstream
```

Comment:
git rebase residual here and some others in this file

Reply:
Thanks @bmeng this not appear when i rebased :0